main.py

```diff
@@ -1,10 +1,56 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    Race.objects.all().delete()
+    Skill.objects.all().delete()
+    Guild.objects.all().delete()
+    Player.objects.all().delete()
+
+    with open("players.json") as f:
+        players = json.load(f)
+        for player in players:
+            race_name = players[player]["race"]["name"]
+            race_description = players[player]["race"]["description"]
+            race, create = Race.objects.get_or_create(
+                name=race_name,
+                description=race_description
+            )
+
+            for skill in players[player]["race"]["skills"]:
+                name = skill["name"]
+                bonus = skill["bonus"]
+                Skill.objects.get_or_create(
+                    name=name,
+                    bonus=bonus,
+                    race_id=race.id
+                )
```

Comment:
There is no need to create additional variables, pass values from the dict directly
```suggestion
                Skill.objects.get_or_create(
                    name=skill["name"],
                    bonus=skill["bonus"],
                    race_id=race.id
                )
```